modules/core/test/test_intrin_utils.hpp

```diff
@@ -1598,24 +1598,25 @@ template<typename R> struct TheTest
         out.a.clear();
         v_store(out.a.d, r1);
         EXPECT_EQ(data.a, out.a);
-
+#endif
         return *this;
     }
+
     TheTest & test_float_cvt_fp16()
     {
+#if CV_SIMD_FP16
         printf("test_float_cvt_fp16 ...\n");
         AlignedData<v_float32> data;
 
         // check conversion
         v_float32 r1 = vx_load(data.a.d);
         v_float16 r2 = v_cvt_f16(r1, vx_setzero_f32());
         v_float32 r3 = v_cvt_f32(r2);
-        EXPECT_EQ(0x3c00, v_get0(r2));
+        EXPECT_EQ(1, v_get0(r2));
```

Comment:
Not sure why `0x3c00` was put here. Value should be `1` on my machine (macOS 14.4.1, M1).